### PR TITLE
feat(docs): microservices be gone

### PR DESCRIPTION
**Pull request by mmomjian**

### CODE REVIEW

File: docs/docs/install/environment-variables.md [hunk 1]
```diff
@@ -166,13 +181,13 @@ Other machine learning parameters can be tuned from the admin UI.
...
-| `IMMICH_API_METRICS`           | Toggle metrics for endpoints and response times (one of [`true`, `false`])                    |         | server, microservices |
-| `IMMICH_HOST_METRICS`          | Toggle metrics for CPU and memory utilization for host and process (one of [`true`, `false`]) |         | server, microservices |
-| `IMMICH_IO_METRICS`            | Toggle metrics for database queries, image processing, etc. (one of [`true`, `false`])        |         | server, microservices |
-| `IMMICH_JOB_METRICS`           | Toggle metrics for jobs and queues (one of [`true`, `false`])                                 |         | server, microservices |
+| Variable                       | Description                                                                                   | Default | Services | Worker             |
+| :----------------------------- | :-------------------------------------------------------------------------------------------- | :-----: | :------- | :----------------- |
+| `IMMICH_METRICS`<sup>\*1</sup> | Toggle all metrics (one of [`true`, `false`])                                                 |         | server   | api, microservices |
```
Comment (mmomjian) [inline docs/docs/install/environment-variables.md:186]: Do the microservices actually use metrics?

Comment (bo0tzz) [reply docs/docs/install/environment-variables.md]: Yes. Speaking of, that's a good point. @zackpollard how does having api and microservices in one container interact with the otel metrics system?

**Comment by mmomjian** [comment]
There are two conflicts that are tricky to fix but I have included all the new changes in this PR, so it should be OK to merge.

**Review by zackpollard — APPROVED** [review]

File: docs/docs/administration/jobs-workers.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+The `immich-server` container contains multiple workers:
+
+- `api`: responds to API requests for data and files for the web and mobile app.
+- `microservices`: handles most other work, such as thumbnail generation and video encoding, in the form of _jobs_. Simply put, a job is a request to process data in the background.
+
+## Split workers
+
+If you prefer to throttle or distribute the workers, you can do this using the [environment variables](/docs/install/environment-variables) to specify which container should pick up which tasks.
+
+For example, for a simple setup with one container for the web UI and one for all other microservices, you can do the following:
```
Comment (zackpollard) [inline docs/docs/administration/jobs-workers.md:16]:
```suggestion
For example, for a simple setup with one container for the Web/API and one for all other microservices, you can do the following:
```

**Comment by mmomjian** [comment]
Looks great, thanks!

**Comment by alextran1502** [comment]
We can merge this, right? @zackpollard

File: docs/docs/administration/jobs-workers.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+services:
+  immich-server:
+    ...
++   environment:
++     IMMICH_WORKERS_INCLUDE: 'api'
+
+  immich-microservices:
+    ...
++   environment:
++     IMMICH_WORKERS_EXCLUDE: 'api'
```
Comment (alextran1502) [inline docs/docs/administration/jobs-workers.md:42]: Shouldn't this be `'microservices'`?

**Pull request merged**